examples/filesystem/README.md

```diff
@@ -7,16 +7,23 @@ application.
 
 In particular, this example shows:
 
-- how to mount/format/unmount a file system, either with spiffs, littlefs, fatfs
-  or constfs
+- how to mount/format/unmount a file system, either with constfs for all
+  devices and littlefs for devices with external flash memory
 - how to open/read/write/close a file with and without newlib
 
+A `constfs` file system is demonstrated with two files for all targets
+to demonstrate the file system functionality without an external mass
+storage device.
+
+For targets that feature an onboard external mass storage device like
+the Nordic nRF52840 Development Kit (see `boards/nrf52840dk`), the 
+flash chip is automatically added by VFS.
 In RIOT, most file systems use a `mtd` as flash interface. So to use this
-example one must define `MTD_0`. `MTD_0` is a pointer to a `mtd_dev_t`
-instance.
+example the define `MTD_0` must exist. `MTD_0` is a pointer to a 
+`mtd_dev_t` instance. 
```

Comment:
We dropped the `MTD_0` defines and the example does not need them.

Reply:
The define with MTD_0 is still present in the `boards/nrf52840dk/include/board.h` file, so I assumed it was still necessary.